src/cpp/benders/benders_core/BendersMathLogger.cpp

```diff
@@ -0,0 +1,77 @@
+#include "BendersMathLogger.h"
+
+#include "LogUtils.h"
+#include "LoggerUtils.h"
+
+HeadersManager::HeadersManager(HEADERSTYPE type, const BENDERSMETHOD& method) {
+  headers_list.push_back("ITE");
+  headers_list.push_back("LB");
+  if (method == BENDERSMETHOD::BENDERS) {
+    headers_list.push_back("UB");
+    headers_list.push_back("BESTUB");
+    headers_list.push_back("AGAP");
+    headers_list.push_back("RGAP");
+  }
+  headers_list.push_back("MinSpx");
+  headers_list.push_back("MaxSpx");
+
+  if (type == HEADERSTYPE::LONG || method == BENDERSMETHOD::BENDERSBYBATCH) {
+    headers_list.push_back("NbSubPbSolv");
+  }
+
+  if (type == HEADERSTYPE::LONG) {
+    headers_list.push_back("CumulNbSubPbSolv");
+  }
+  headers_list.push_back("IteTime");
+  headers_list.push_back("MasterTime");
+  headers_list.push_back("SPWallTime");
+  if (type == HEADERSTYPE::LONG) {
+    headers_list.push_back("SPCpuTime");
+    headers_list.push_back("NotSolvingWallTime");
```

Comment:
Add unit :
- `IteTime (s)`
- `MasterTime (s)`
- `SPWallTime (s)`
- `SPCpuTime (s)`
- `NotSolvingWallTime (s)`
I think it should still be ok regarding of the lengths of the headers

Reply:
Why have you closed this thread ? Headers too long ?

Reply:
ok i updated!
and yes it's too long, they are all  seconds and introducing whitespace is a bit annoying for ex reading the results file with a spreadsheet editor etc...